webpack/scenes/Subscriptions/Details/index.js

```diff
@@ -8,6 +8,7 @@ import SubscriptionDetails from './SubscriptionDetails';
 // map state to props
 const mapStateToProps = state => ({
   subscriptionDetails: state.katello.subscriptionDetails,
+  enabledProducts: state.katello.enabledProducts,
```

Comment:
I'm wondering if we should make` enabledProducts` part of `subscriptionDetails`? This may make more sense as the products will be related to the subscription information, though I haven't looked at an example response yet.

Reply:
it actually is put on the subscriptionDetails. I guess this line isn't needed